store/heightsub.go

```diff
@@ -34,97 +39,86 @@ func (hs *heightSub[H]) Height() uint64 {
 }
 
 // SetHeight sets the new head height for heightSub.
+// Unblocks all awaiting [Wait] calls in range from [heightSub.Height] to height.
 func (hs *heightSub[H]) SetHeight(height uint64) {
```

Comment:
SetHeight doesn't seem to be used anymore, is this expected?

Reply:
Woow, no. It's used, see `Store.Init` and `Store.Head`.

Reply:
After refactor of `UnblockHeight` I found a bug where `SetHeight` must be used instead (yep, in `updateContiguousHead`).

Reply:
Yes, i was thinking about advancing usage in the first place